src/screens/Podcast/common/constants/index.ts

```diff
@@ -0,0 +1,9 @@
+const DEFAULT_EPISODES_OFFSET = 0
+const DEFAULT_EPISODES_LIMIT = 5
```

Comment:
We can avoid this constants

Reply:
I trying do it, but there i need one of this constants DEFAULT_EPISODES_LIMIT:
src/screens/Podcast/Podcast.tsx => line number 56 and in this case i decided add constants for another value DEFAULT_EPISODES_OFFSET

Reply:
Okay, I see.